TYPO3.Neos/Resources/Public/JavaScript/Content/Components/TargetWorkspaceController.js

```diff
@@ -156,6 +156,8 @@ define(
 				HttpRestClient.updateResource('neos-service-workspaces', this.get('userWorkspace.name'), options).then(function(result) {
 					that._loadConfiguration();
 					ContentModule.reloadPage();
+					EventDispatcher.trigger('nodesInvalidated');
+					EventDispatcher.trigger('contentChanged');
```

Comment:
Nope, `nodesInvalidated` will trigger the nodetree to reload, `contentChanged` the structureTree. None of those events triggered the structure and the node tree. Or is there another event that could be used @aertmann ?